docs/rum_mobile_vitals.md

```diff
@@ -0,0 +1,41 @@
+
+| Measurement                    | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+|--------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Slow renders                   | To ensure a smooth, [jank-free][5] user experience, your application should render frames in under 60 seconds. <br /><br />  RUM tracks the application’s [display refresh rate][6] using `@view.refresh_rate_average` and `@view.refresh_rate_min` view attributes. <br /><br />  With slow rendering, you can monitor which views are taking longer than 40, 50, or 60 seconds to render. |
+| Frozen frames                  | Frames that take longer than 700ms to render appear as stuck and unresponsive in your application. These are classified as [frozen frames][7]. <br /><br />  RUM tracks `long task` events with the duration for any task taking longer then 100ms to complete. <br /><br />  With frozen frames, you can monitor which views appear frozen (taking longer than 700ms to render) to your end users and eliminate jank in your application.                                                                                                                                                                                                 |
+| Application not responding     | When the UI thread of an application is blocked for more than 5 seconds, an `Application Not Responding` ([ANR][8]) error triggers. If the application is in the foreground, the system displays a dialog modal to the user, allowing them to force quit the application. <br /><br />   RUM tracks the ANR rate over time and captures the entire error report when an ANR error is encountered.                                                                                                                                                                                                                              |
```

Comment:
> RUM tracks the ANR rate over time and 

ANR are not rates but an event that can occur. Instead we might say : 
> RUM tracks ANR occurrences and captures the entire stack trace blocking the main thread when an ANR is encountered

Reply:
```suggestion
| Application not responding     | When the UI thread of an application is blocked for more than 5 seconds, an `Application Not Responding` ([ANR][8]) error triggers. If the application is in the foreground, the system displays a dialog modal to the user, allowing them to force quit the application. <br /><br />   RUM tracks ANR occurrences and captures the entire stack trace that blocks the main thread when it encounters an ANR.                                                                                                                                                                                                                              |
```